303-quarkus-vertx-sql/src/test/resources/application.properties

```diff
@@ -8,11 +8,13 @@ quarkus.http.test.port=8081
 # Postgresql
 # Database
 %postgresql.app.selected.db=postgresql
+%postgresql.quarkus.flyway.mysql.migrate-at-start=false
+%postgresql.quarkus.flyway.db2.migrate-at-start=false
 quarkus.datasource.db-kind=postgresql
 quarkus.datasource.username=test
 quarkus.datasource.password=test
-quarkus.datasource.reactive.url=postgresql://localhost:5432/amadeus
-quarkus.datasource.jdbc.url=jdbc:postgresql://localhost:5432/amadeus
+quarkus.datasource.reactive.url=${postgres.url}
+quarkus.datasource.jdbc.url=jdbc:${postgres.url}
```

Comment:
Could the application be launched ( `mvn quarkus:dev` )  with these properties?.

Reply:
`quarkus:dev` use the properties file in `src/main/resources`, not from the test resources.